source/PlayerInfoPanel.cpp

```diff
@@ -505,8 +506,16 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 		table.Advance();
 		table.DrawGap(5);
 		
-		table.DrawTruncatedPair(combatRating, dim,
-			"(" + to_string(combatLevel) + ")", dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("rank:", dim,
+			to_string(combatLevel), dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("title:", dim,
+			combatRating, dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("experience:", dim,
+			Format::Number(combatExperience), dim, Truncate::MIDDLE, false);
+		bool maxRank = (combatRating == GameData::Rating("combat", combatLevel + 1));
```

Comment:
I will say that it's technically possible for us to have two ranks in a row with the same title, which under this line would cause the experience to the next rank to display as MAX, so a better check for max rank might be to see if `experience >= 2000000000`, which is the max rating as defined by PlayerInfo::HandleEvent.

Reply:
I set it up that way in case someone modifies the list of titles to have less/more (if possible) titles than vanilla. If we changed it to that number, and someone had a list with less titles, the game would keep going "New title coming up!" and not give anything.

Both are non-vanilla modifications, and especially if we're removing rank #, I'd just want to base it on someone who changed the number of titles, rather someone who theoretically puts two identical titles in a row before getting new ones.